### ISSUE TITLE
Show Posts as Read

### ISSUE THREAD

**Issue by NonITExpert** [hariettem/Artemis#156]
Android: I don't hide my read posts, instead I need to see which ones I’ve previously opened/read. I'd like to see the option to choose either 'Show as Read' or 'Hide Read Posts'.  This is done well in Infinity for Reddit. I’ve attached screenshot of a menu in settings, as well as Post list view.

![Screenshot_20230703_141027_Infinity](https://github.com/hariettem/Artemis/assets/138516886/af1e1bd4-34eb-4d8c-9c1b-fb7187ccd612)

![Screenshot_20230703_140256_Infinity](https://github.com/hariettem/Artemis/assets/138516886/95e6e419-8eed-4b3d-8b3d-43a2e696bafd)

**Comment by Alaric323** [comment]
This was previously mentioned in the discord as an idea; Sync for Reddit dulled the text color (white to grey) of read posts which I think would work well here, but certain themes may cause issues.

**Comment by NonITExpert** [comment]
Yeah that could work. Can we make sure the contrast between read/unread is a noticeable one?  This will assist users who are eyesight challenged.

**Comment by hariettem** [comment]
I like marking as read after voting. will add that bit

**Comment by NonITExpert** [comment]
I orefer mark as read after opening the post as well.

**Comment by NonITExpert** [comment]
Sometimes mark as read after opening doesn't work. Watch what happens in the clip with post titled Recap of the Lemmy XSS Incident...
https://github.com/hariettem/Artemis/assets/138516886/d37abaee-6272-4b34-b4c3-d2738cf58aad

**Comment by Jay2645** [comment]
Relay for Reddit had this, too. 

If you had an OLED theme, read posts would become an off-black:

![Screenshot_20230712-014059](https://github.com/hariettem/Artemis/assets/2058763/75c7f42e-e181-4931-a005-36fa256b0868)

Any other theme would adopt a color from the theme itself for read posts (this is the blue theme):

![Screenshot_20230712-014121](https://github.com/hariettem/Artemis/assets/2058763/d199f3bf-ea07-487d-8941-ea659f064f7f)

You can press the big button in the bottom center to hide anything that has been read. This button actually hid on the account level, I believe - I wouldn't see those posts on the desktop either.

I used this feature heavily for years, but over time I actually moved to enabling the Reddit-wide setting for hiding posts that I've voted on. Then any posts I voted on wouldn't appear in my main feed at all (I think this happened on an API level).

I'd be happy with either hiding on "read" or "voted" or both. Before I had access to Artemis, I used a Firefox PWA with a Tampermonkey script that hid posts after I voted. But I thought I'd add my 2 cents. ;)

**Comment by NonITExpert** [comment]
Please add option to 'mark as read' after viewing post’s article or image.
